🌐 Add 'third-world country' (#251)
selfdefined/web-app: 11ty/definitions/third-world-country.md

---
title: third-world country
slug: third-world-country
defined: true

> speech: noun

Reviewer: `
```suggestion
speech: noun
flag:
  level: avoid
  warning: colonialist language

```

---

> a problematic term used to refer to a country that doesn’t quite align with Western ideology or the Western image of society. (_Also common are terms such as “third-world” (adj.), “first-world”, “developing/developed nations”, etc._)

Reviewer: ```suggestion
describes a country that doesn’t quite align with Western ideology or the Western image of society. (_Also common are terms such as “third-world” (adj.), “first-world”, “developing/developed nations”, etc._)
```

Reviewer: We usually try to omit the "qualifications" of a word (i.e., if it's problematic) until the issues section (though you could probably find examples that were inconsistent with this rule 😅) until we've provided the sterilised description of the word. This edit also removes the self-referential "term used to..."

Author: Totally makes sense — thank you for this!

## History

> “Three worlds” terminology originally arose during the cold war to define countries that were aligned with NATO (“1st world”), aligned with the Warsaw Pact (“2nd world”), or non-aligned (“3rd world”). Because many third-world countries were economically poor, it became common to refer to poor countries as third-world.

Reviewer: ```suggestion
“Three worlds” terminology originally arose during the cold war to define countries that were aligned with North Atlantic Treaty Organization (NATO) (“1st world”), aligned with the Warsaw Pact (“2nd world”), or non-aligned (“3rd world”). Because many third-world countries were economically poor, it became common to refer to poor countries as third-world.
```

Reviewer: I think some additional insight here as to which countries were listed in each of those three designations would help to reiterate the point.

## Issues

First off, the term is archaic and refers to a /_NATO vs The Communists_/ worldview that is no longer relevant. Secondly, and more importantly, it presupposes a hierarchy between countries, ranked by Western ideals and homogeneity. Western ideology is a fairly narrow way to perceive global socioeconomics. In the words of [Zeeshan Aleem](_https://www.mic.com/articles/107686/why-you-shouldn-t-call-poor-nations-third-world-countries_): “Social democracy in Scandinavia, oil-funded theocracy in Saudi Arabia, and a one-party, partially planned, partially free market economy in China are all vastly different models for generating and harnessing prosperity.”

> 

Reviewer: I think an Impact section here would be helpful.

Some ideas for impact (non-exhaustive):
- Imposes the Western, American/Anglo-centric view of the world
- Misrepresents entire countries and regions as less-than, which dehumanises its people and its progress, reinforcing false stereotypes

Author: Absolutely! I'll commit your suggested changes and work on an Impact section.

## Usage Tip

> Be more specific or contextual. If we mean “countries with high infant mortality rates”, or “countries experiencing an HIV/AIDS outbreak”, we probably ought to say that. It’s much more clear to be specific.

Reviewer: I would also add here that every assessment should be contextualised with the person writing. Even if we avoid the words "third world," are we still superimposing Westernised or colonialist ideas of success within the piece itself? (This becomes easier to understand, I think, when we looked at "developed." Developed has a positive connotation in the Western world, and is defined by Western ideals.
